### ISSUE TITLE
Missing release tag v1.0.5 (12-February-2021)

### ISSUE THREAD

**Issue by rexut** [STMicroelectronics/otm8009a#1]
**Describe the set-up**

N/A - it's a poor release management issue on STMicroelectronics side.

**Describe the bug (skip if none)**

The latest release v1.0.5 for generation V1 BSP's is not reflected by this repository in branch `bspv1`.

**How to reproduce the bug (skip if none)**

Investigate the code base 😄 .

**Additional context**

The following chane is missing:

- Set the PWM backlight frequency to over 22KHz to avoid audible noise.
```
diff --git a/otm8009a.c b/otm8009a.c
index b2fbb38..f2183d2 100644
--- a/otm8009a.c
+++ b/otm8009a.c
@@ -152,7 +152,7 @@ const uint8_t ShortRegData47[] = {0xC5, 0x66};
 const uint8_t ShortRegData48[] = {OTM8009A_CMD_NOP, 0xB6};
 const uint8_t ShortRegData49[] = {0xF5, 0x06};
 const uint8_t ShortRegData50[] = {OTM8009A_CMD_NOP, 0xB1};
-const uint8_t ShortRegData51[] = {0xC6, 0x06};
+const uint8_t ShortRegData51[] = {0xC6, 0x05};
 /**
   * @}
   */
@@ -353,7 +353,7 @@ uint8_t OTM8009A_Init(uint32_t ColorCoding, uint32_t orientation)
   DSI_IO_WriteCmd(0, (uint8_t *)ShortRegData49);
   /////////////////////////////////////////////////////////////////////////////
 
-  /* CABC LEDPWM frequency adjusted to 19,5kHz */
+  /* CABC LEDPWM frequency adjusted to 22,7kHz */
   DSI_IO_WriteCmd(0, (uint8_t *)ShortRegData50);
   DSI_IO_WriteCmd(0, (uint8_t *)ShortRegData51);
```

**Screenshots**

N/A

**Comment by ALABSTM** [comment]
Hi @rexut,

Thank you for this other one. You're right too. Tag `v1.0.5` will be published soon. I'll keep you posted.

With regards,

**Comment by ALABSTM** [comment]
Fixed in commit b7b5e015dbb668e8800b222995cb9370d928db91
